decidim/decidim
# Fix admin language selector with more than 4 locales

#### :tophat: What? Why?
A few issues in admin panel when using more than 4 languages with questionnaires (decidim-forms):
- language selector doesn't work for newly created fields.
- the field disappears when there are errors in the form and you select a different language for this field.
- some translations of language with error are not correct

First issue is that when you add 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes https://github.com/decidim/decidim/issues/9105

#### Testing
Prerequisites:
- in the /config/initializers/decidim.rb, the `config.available_locales` should contain more than 4 languages, so language selector will change from "tabs" style to "select" style.
- login as admin
- choose the site language that is different from default one for organization
- create a Survey component for any assembly/process or whatever
- try to add a new question to that survey and fill in all languages - here is issue 1
- try to save, page will refresh with error (because you didn't fill the default language for a question)
- try to change the language for any field for new question, the field disappears - here is issue 2
- you can also notice that there is a wrong label for language with error for some languages (they mention "English" instead of the language itself).

### :camera: Screenshots
**[Video explaining all 3 issues](https://www.awesomescreenshot.com/video/9823031?key=8b8120abcdbd32f1e94fb12bb9aa002d)**
On that video the default language of the organization is `:fr`!
